### ISSUE TITLE
VeSTige preset left and right arrows are gone

### ISSUE THREAD

**Issue by karmux** [LMMS/lmms#3498]
In VeSTige instrument left and right arrows to browse presets are missing. Instead it shows something that looks like dead pixels. I remember that I was using these arrows about month ago on master branch.

![screenshot_20170411_004313](https://cloud.githubusercontent.com/assets/2575235/24884144/cfe69778-1e50-11e7-8684-089fd43643a5.png)

This picture from Wiki shows how it used to look:

![](https://lmms.io/wiki/images/8/8d/Plugins_VeSTige1.0.0.png)

LMMS 1.2.0-rc2.144
(Linux x86_64, Qt 5.6.1, GCC 6.2.0 20161005)

**Comment by BaraMGB** [comment]
Those pixels appears if a pixmap couldn't be found.

**Comment by karmux** [comment]
@Lukas-W , when I revert 9f905bce3eee4e24a2338d9d8a104af6bdbec8ad I get icons back.
After looking into it a bit I have no idea why some certain pixmaps always get missing.

**Comment by tresf** [comment]
@karmux please see this comment... https://github.com/LMMS/lmms/pull/1891#issuecomment-289303030

**Comment by karmux** [comment]
@tresf but vestige has no underscore in the name, nor has stepper-left.png. I can make stepper-left pixmap visible if I use it for "Show/hide GUI" button instead of zoom pixmap.

**Comment by tresf** [comment]
@karmux good call.  What if you make this line match the others (such as in vibed):

[vestige](https://github.com/LMMS/lmms/blob/9f905bce3eee4e24a2338d9d8a104af6bdbec8ad/plugins/vestige/CMakeLists.txt#L12)
```cmake
BUILD_PLUGIN(vestige EMBEDDED_RESOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.png")
```

[vibed](https://github.com/LMMS/lmms/blob/9f905bce3eee4e24a2338d9d8a104af6bdbec8ad/plugins/vibed/CMakeLists.txt#L3)
```cmake
BUILD_PLUGIN(vibedstrings [...] EMBEDDED_RESOURCES *.png)
```

**Comment by tresf** [comment]
Scratch that... that won't change anything and these don't exist in that directory anyway.  These appear to be shared artwork.  Still hunting...

**Comment by tresf** [comment]
Ok... I think we need to replace [`PLUGIN_NAME::` with `embed` for shared LMMS (non-plugin) resources](https://github.com/LMMS/lmms/blob/9f905bce3eee4e24a2338d9d8a104af6bdbec8ad/plugins/vestige/vestige.cpp#L444-L447).  @Lukas-W should confirm...

**Comment by karmux** [comment]
I got all pixmaps back by replacing `PLUGIN_NAME::getIconPixmap` with `embed::getIconPixmap` in vestige.cpp only for missing pixmaps. Replacing every pixmap made other pixmaps missing.

state_stop icon actually seems like missing (also on Wiki image).

**Comment by karmux** [comment]
VeSTige in #3501.

![screenshot_20170412_202826](https://cloud.githubusercontent.com/assets/2575235/24970877/9af9406e-1fbe-11e7-9f90-11c25c0380cf.png)

**Comment by karmux** [comment]
Fixed in #3501.
